integreat_compass/cms/models/offers/offer.py

```diff
@@ -40,7 +40,7 @@ def public_version(self):
         :return: OfferVersion or ``None``
         :rtype: ~integreat_compass.cms.models.offers.offer_version.OfferVersion
         """
-        versions = self.versions.order_by("offer_version_date")
+        versions = self.versions.order_by("created_at")
         for version in versions:
```

Comment:
```suggestion
        for version in self.versions.all():
```
Ordering is no longer required, since this already happens in the meta class of the model (which automatically applies the ordering to all querysets, including related managers).